src/resources/Resource.ts

```diff
@@ -87,9 +87,9 @@ export default class Resource<R, T extends R> {
         }
         return this.injectPrototypes(response.data);
       },
-      delete: async <S extends T | true>(url: string): Promise<S> => {
+      delete: async <S extends T | true>(url: string, context: any): Promise<S> => {
```

Comment:
Does this need a default value?

Reply:
We could do `undefined` by default. I'm OK with that.

Reply:
It's also fine with leaving it like this.

Reply:
I thought about it a little more, and now I like the idea a little more. I just changed it.